tests/unit/hl7/test_fhir.py

```diff
@@ -36,6 +36,22 @@ def test_fhir_record_to_pii_record():
                         "code" : "SS"
                     }]
                 },
+            },
```

Comment:
I think the payload would look a bit more like the following
```
  {
    "use": "official",
    "type": {
      "coding": [
        {
          "system": "http://terminology.hl7.org/CodeSystem/v2-0203",
          "code": "DL",
          "display": "Driver's License"
        }
      ],
      "text": "Driver's License"
    },
    "system": "urn:oid:2.16.840.1.113883.3.19.3.1.8",
    "value": "D1234567",
    "assigner": {
      "display": "California Department of Motor Vehicles",
      "identifier": {
        "system": "urn:oid:2.16.840.1.113883.19.3.1.1", 
        "value": "CA"
      }
    }
  }
```

Reply:
I think so, I don't think the display name is going to be consistent.  I think sometimes we might just see "DMV" or "Department of Motor Vehicles", in which case we won't know what state issued the license.  From the FHIR docs, display is just "Text alternative for the resource"

Reply:
Added this payload to the test and updated the parser.